examples/generatesvg.go

```diff
@@ -18,7 +18,7 @@ import (
 // iconURL takes a reference to a charm and returns the URL for that charm's icon.
 // In this case, we're using the api.jujucharms.com API to provide the icon's URL.
 func iconURL(ref *charm.Reference) string {
-	return "https://api.jujucharms.com/v4/" + ref.Path() + "/archive/icon.svg"
+	return "https://api.jujucharms.com/v4/" + ref.Path() + "/icon.svg"
```

Comment:
If you're going to use the proper endpoint it should be https://api.jujucharms.com/charmstore/v4/...